corehq/apps/userreports/pillow.py

```diff
@@ -70,19 +70,29 @@ def rebuild_tables_if_necessary(self):
                 migration_context = get_migration_context(connection, table_map.keys())
                 diffs = compare_metadata(migration_context, metadata)
 
-            tables_to_rebuild = get_tables_to_rebuild(diffs, table_map.keys())
+            tables_to_rebuild = set(get_tables_to_rebuild(diffs, table_map.keys()))
```

Comment:
locally, this was rebuilding the same table 5 times when I made a change before adding this